Library/Formula/go.rb

```diff
@@ -8,7 +8,7 @@ class Go < Formula
   sha1 "460caac03379f746c473814a65223397e9c9a2f6"
   version "1.4.2"
 
-  head "https://go.googlesource.com/go", :using => :git
+  head "https://github.com/golang/go", :using => :git
```

Comment:
change the url to `https://github.com/golang/go.git` and you'll be able to drop the `:using => :git`